internal/consensus/common_test.go

```diff
@@ -151,6 +154,12 @@ func signVote(
 		panic(fmt.Errorf("failed to sign vote: %v", err))
 	}
 
+	// TODO: remove hardcoded vote extension.
```

Comment:
will this be removed in this PR or a following?

Reply:
Removed!